css/index.css

```
@@ -104,14 +104,116 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
+    background: teal;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.middle-content .boxes .box2 {
+    width: 12.5%;
+    height: 100px;
+    background: gold;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.middle-content .boxes .box3{
```

So as we discussed during our 1 on 1 time, it would have been better to place all the properties that you're using over and over, i.e.;
```suggestion
    width: 12.5%;
    height: 100px;
    margin: 20px 2.5%;
    color: white;
    display: flex;
    align-items: center;
    justify-content: center;
```
Into one class, and then only the property(ies) that are changing in the more specific class. I.e.; the background in this case.